client/src/App.js

```diff
@@ -49,14 +49,41 @@ export default () => {
     signupDisclosure.onClose();
 
     // TODO: Make fetch() request here to signup
-
-    loginDisclosure.onOpen();
+    fetch('http://localhost:5000/signup', {
```

Comment:
You're going to be using fetch to this URL with the same headers, mode, and JSON.stringify call quite a bit. Maybe you should abstract that into a function where you just supply the endpoint, body, and method?  Something like:

```js
api.post('/signup', { email: 'xxx', password: 'yyy' });
```